kratos/tests/test_matrix_interface.py

```diff
@@ -101,5 +101,17 @@ def test_scalar_prod(self):
         A *= 2.0
         self.assertMatrixAlmostEqual(C, A)
 
+    def test_matrix_transpose(self):
+        A = KM.Matrix(2,3)
+        B = KM.Matrix(3,2)
+        for i in range(A.Size1()):
+            for j in range(A.Size2()):
+                A[i,j] = i
+        B = A.transpose()
+
+        for i in range(A.Size1()):
+            for j in range(A.Size2()):
+                self.assertEqual(A[i,j], B[j,i])

```

Comment:
```suggestion
        self.assertMatrixAlmostEqual(A, B)
```

Reply:
the matrices are not equal as i and j are different

Reply:
sure
:man_facepalming: